src/processes/round.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "round",
+  "summary": "Rounds to a specified precision",
+  "description": "Rounds a real number `x` to specified precision `p`.\n\nIf the fractional part of `x` is halfway between two integers, one of which is even and the other odd, then the even number is returned.\nThis behaviour follows [IEEE Standard 754](https://ieeexplore.ieee.org/document/4610935). This kind of rounding is also called \"rounding to nearest\" or \"banker's rounding\". It minimizes rounding errors that result from consistently rounding a midpoint value in a single direction.\n\nThe no-data value `null` is passed through and therefore gets propagated.",
+  "categories": [
+    "math > rounding"
+  ],
+  "parameter_order": [
```

Comment:
You removed p, but it's still mentioned in the description and examples. I think we can simply implement p ourself if it's not supported by Google directly. Just with some math...